### PR TITLE
Add CelebA as a proper EthicML dataset

### PR DESCRIPTION
**Pull request by thomkeh**

### CODE REVIEW

File: ethicml/data/health.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def name(self) -> str:
     @property
     def filename(self) -> str:
         """Getter for filename."""
-        return "health.csv"
+        return "health.csv.zip"
```
Comment (olliethomas) [inline ethicml/data/health.py]: ???

Comment (olliethomas) [inline ethicml/data/health.py]:
```suggestion
        return "health.csv"
```

**Comment by thomkeh** [comment]
You don't like the use of zipped CSV files?

**Comment by olliethomas** [comment]
TBH, I thought that you'd run find and replace on .csv to .csv.zip. I didn't realise that all csvs had been zipped

**Pull request merged**